fe/topology.py

```diff
@@ -375,11 +373,20 @@ def parameterize_harmonic_bond(self, ff_params):
     def parameterize_harmonic_angle(self, ff_params):
         return self._parameterize_bonded_term(ff_params, self.ff.ha_handle, potentials.HarmonicAngle)
 
+    def parameterize_periodic_torsion(self, proper_params, improper_params):
+        """
+        Parameterize all periodic torsions in the system.
+        """
+        proper_params, proper_potential = self.parameterize_proper_torsion(proper_params)
+        improper_params, improper_potential = self.parameterize_improper_torsion(improper_params)
+        combined_params = jnp.concatenate([proper_params, improper_params])
+        combined_idxs = np.concatenate([proper_potential.get_idxs(), improper_potential.get_idxs()])
+        combined_potential = potentials.PeriodicTorsion(combined_idxs)
+        return combined_params, combined_potential
```

Comment:
Confirming if this is part of the current PR?

Reply:
yes - I forgot to add this for the DualTopology class